subxt/src/tx/tx_client.rs

```diff
@@ -425,6 +429,15 @@ where
     pub fn into_encoded(self) -> Vec<u8> {
         self.encoded.0
     }
+
+    /// Returns the SCALE encoded extrinsic bytes with the length attached to the end.
+    /// Useful for certain RPC-calls that expect this format.
+    ///
+    /// *Example*: let self.encoded.0 be [10, 20, 10, 20, 10], then encoded_with_len() will be [10, 20, 10, 20, 10, 5, 0, 0, 0]. So there are always 4 bytes appended representing the length of the extrinsic in bytes in little endian notation.
```

Comment:
would be better with a `doc test` or `doc example` instead this long line very hard to read